ops/__init__.py

```diff
@@ -12,9 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""The Operator Framework."""
+"""The Charmed Operator Framework.
 
-from .version import version as __version__  # noqa: F401 (imported but unused)
+The Charmed Operator Framework allows the development of operators in a simple
+and straightforward way, using standard Python structures to allow for clean,
+maintainable, and reusable code.
+
+A Kubernetes operator is a container that drives lifecycle management,
+configuration, integration and daily actions for an application. Operators
+simplify software management and operations. They capture reusable app domain
+knowledge from experts in a software component that can be shared.
+
+The Charmed Operator Framework extends the "operator pattern" to enable Charmed
+Operators, packaged as and often referred to as “charms”. Charms are not just
```

Comment:
Nit: should use normal ASCII `"` quote here (like on the line above), not smart quotes.

Reply:
Agreed, not sure how they slipped in there. Thanks!